src/inmanta/module.py

```diff
@@ -2300,6 +2297,20 @@ def __init__(self, project: Optional[Project], path: str):
             super(ModuleV1, self).__init__(project, path)
         except InvalidMetadata as e:
             raise InvalidModuleException(f"The module found at {path} is not a valid V1 module") from e
+        except ModuleMetadataFileNotFound:
+            if os.path.exists(os.path.join(path, ModuleV2.MODULE_FILE)):
+                raise ModuleMetadataFileNotFound(
+                    f"Module at {path} looks like a v2 module. Please add it as a v2 requirement with"
+                    " `inmanta modules add <module_name>`"
```

Comment:
@arnaudsjs is this correct or does it require a `--v2`

Reply:
Good point, I don't think this is correct.